### PR TITLE
change uniq to distinct for SQL queries to fix deprecation

### PR DESCRIPTION
**Pull request by sspsk**

### CODE REVIEW

File: app/controllers/admin/versions_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def index
       if current_user.has_role? :organization_admin, :any
         @conferences_with_role = Organization.with_role('organization_admin', current_user).map { |org| org.conferences.pluck :short_title }.flatten
       end
-      @conferences_with_role.uniq!
+      @conferences_with_role.distinct!
```
Comment (differentreality) [inline app/controllers/admin/versions_controller.rb]: In line 10, `.map` returns an array, so this shouldn't be changed.

**Comment by differentreality** [comment]
@sspsk please make sure that Travis has no errors for the lines of code you are changing, so that we can review and merge the PR :)

**Comment by differentreality** [comment]
@sspsk do you need help to proceed with this PR?

**Comment by hennevogel** [comment]
I guess not. @sspsk please open another PR if you're up for working on this

**Pull request closed without merging**